app/controllers/AnalyticsController.scala

```diff
@@ -6,16 +6,15 @@ import play.api.libs.json.JsObject
 import play.api.mvc.{Action, PlayBodyParsers}
 import play.silhouette.api.Silhouette
 import security.WkEnv
-import utils.{ApiVersioning, WkConf}
+import utils.WkConf
 
 import javax.inject.Inject
 import scala.concurrent.ExecutionContext
 
 class AnalyticsController @Inject()(analyticsService: AnalyticsService, conf: WkConf, sil: Silhouette[WkEnv])(
     implicit ec: ExecutionContext,
     bodyParsers: PlayBodyParsers)
-    extends Controller
-    with ApiVersioning {
+    extends Controller {
 
```

Comment:
Here is the missing comment: Just double checking: You removed the `ApiVersioning` inheritance here as it was unused?